SPECS/rubygem-td-client/rubygem-td-client.spec

```diff
@@ -0,0 +1,37 @@
+%global debug_package %{nil}
+%global gemdir %(IFS=: R=($(gem env gempath)); echo ${R[${#R[@]}-1]})
+%global gem_name td-client
+Summary:        Ruby Client Library for Treasure Data
+Name:           rubygem-td-client
+Version:        1.0.7
+Release:        1%{?dist}
+License:        Apache 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages
+URL:            https://rubygems.org/gems/%{gem_name}/versions/%{version}
+Source0:        https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+BuildRequires:  ruby >= 2.1.0
+Requires:       rubygem-httpclient >= 2.7
+Requires:       rubygem-msgpack < 2
+Requires:       rubygem-msgpack >= 0.5.6
```

Comment:
Do we even offer versions that would be conflicting?  Can we simplify?

Reply:
Resolved